helpers/remote.py

```diff
@@ -1,100 +1,181 @@
 """
-Controls node over SSH if remote, or via OS if local one.
-The API is required to transparently handle both the cases - Tempesta and the
-framework on the same node (developer tests case) or on separate machines (CI
-case).
+Module to work with nodes: local and remote(via SSH).
+
+The API is required to transparently handle both cases - Tempesta and the framework
+on the same node (developer test case), or on separate machines (CI case).
 """
 from __future__ import print_function
 
 import abc
 import errno
+import logging
 import os
 import re
-import shutil
 import subprocess
 import time
-from dataclasses import dataclass
-from typing import Union  # TODO: use | instead when we move to python3.10
+from typing import Any, Optional
 
 import paramiko
 
 import run_config
 
-from . import error, tf_cfg
+from helpers import error, tf_cfg
 
 __author__ = "Tempesta Technologies, Inc."
-__copyright__ = "Copyright (C) 2017 Tempesta Technologies, Inc."
+__copyright__ = "Copyright (C) 2024 Tempesta Technologies, Inc."
 __license__ = "GPL2"
 
+LOGGER = logging.getLogger(__name__)
+
 # Don't remove files from remote node. Helpful for tests development.
 DEBUG_FILES = False
 # Default timeout for SSH sessions and command processing.
 DEFAULT_TIMEOUT = 10
 
 
-class Node(object, metaclass=abc.ABCMeta):
-    def __init__(self, type, hostname, workdir):
+class INode(object, metaclass=abc.ABCMeta):
+    """Node interfaces."""
+
+    LOGGER = LOGGER
```

Comment:
I work with the logging because: I reworked almost a whole module, and we worked with the new logger before, and you are making changes for the new logging, so I decided to make logger-related updates because we already configured new logger and we may easily use it whenever we want. So, to not come back to the module again.
One more, we can easily skip using `tf_cfg.dbg` because we already configured the logger (I made one small update to allow to do so, actually, I fixed a mistake in the configuration).

Reply:
https://github.com/tempesta-tech/tempesta-test/pull/696#discussion_r1806092029

Reply:
We discussed everything over the phone, changes will be applied with the next commit.